engine/table/src/main/java/io/deephaven/engine/util/jpy/JpyInit.java

```diff
@@ -64,4 +65,9 @@ private static void markJvmReady() {
             // empty
         }
     }
+
+    private static void checkPythonEnv() {
+        final PyModule deephavenJpyModule = PyModule.importModule("deephaven_internal.jvm");
+        final PyObject obj = deephavenJpyModule.callMethod("check_py_env");
+    }
```

Comment:
This might be obsolete given my advice on creating a single entrypoint; but we should preferably try/close PyModule / PyObject calls like this.